recipes/setup.rb

```diff
@@ -58,7 +58,7 @@
   # dependencies
   execute "dependencies #{repo}" do
     action  :nothing
-    command 'scripts/dependencies.sh'
+    command "scripts/dependencies.sh #{node[:valhalla][:src_dir]}"
```

Comment:
this tells tyr when it installs to build all of its deps (including mjolnir) in mnt/valhalla/src